Gravity/Gravity.Test.Unit/RdoExtensionsTests.cs

```diff
@@ -0,0 +1,192 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Linq.Expressions;
+using System.Reflection;
+using Gravity.Base;
+using Gravity.Extensions;
+using Gravity.Test.TestClasses;
+using kCura.Relativity.Client.DTOs;
+using NUnit.Framework;
+using NUnit.Framework.Constraints;
+
+namespace Gravity.Test.Unit
+{
+	public class RdoExtensionsTests
+	{
+		[Test]
+		[TestCaseSource(nameof(ToHydratedDto_SimpleTypesFields_TestCases))]
+		public void ToHydratedDto_SimpleTypesFields(string fieldName, object value)
+		{
+			var property = typeof(GravityLevelOne).GetProperty(fieldName);
+			var propertyGuid = property.GetCustomAttribute<RelativityObjectFieldAttribute>().FieldGuid;
+
+			//make sure what you put into the RDO gets copied to the DTO
+			var dto = GetRdoWithField(propertyGuid, new FieldValue() { Value = value });
+
+			Assert.AreEqual(value, property.GetValue(dto));
+		}
+
+		public static IEnumerable<TestCaseData> ToHydratedDto_SimpleTypesFields_TestCases()
```

Comment:
Let's talk about this. I did it a little differently in the Integration Test.  I created a TestDefinition class and referenced it differently.   
`[Test, Description("Verify RelativityObject field created correctly using Gravity"),      TestCaseSource(typeof(TestCaseDefinition), "SimpleFieldReadWriteTestCases")]`

Not sure it really matters we should just be consistent.

Reply:
I think the principle of locality says we should keep them in the same class as they are needed. Are you OK with that?

Reply:
I am OK with it.  I will look an incorporating this style in the other tests.